pkg/manager/manager.go

```diff
@@ -289,17 +290,82 @@ func (m *manager) AttachRepresentor(conf *types.PluginConf) error {
 		if err = utils.BridgeTrunkVlanAdd(m.nLink, rep, conf.Trunk); err != nil {
 			return fmt.Errorf("failed to add trunk VLAN for representor %s: %v", conf.Representor, err)
 		}
+		log.Info().Msgf("Setting multiple VLANs for rep %s: %v", conf.Representor, conf.Trunk)
```

Comment:
please move the log to the begning of this block

Reply:
done